tests/post/steps/conftest.py

```diff
@@ -6,3 +6,12 @@ def check_service(host):
        cmd = "kubectl --kubeconfig=/etc/kubernetes/admin.conf cluster-info"
        retcode = host.run(cmd).rc
        assert retcode == 0
+
+@given(parsers.parse("Pods with app label '{app}' are '{state}'"))
+def check_pod_state(host, app, state):
+    with host.sudo():
+        cmd = ("kubectl --kubeconfig=/etc/kubernetes/admin.conf"
```

Comment:
Could be good to have a wrapper around this command and this configuration file since commands are quite long.